Packs/rasterize/ReleaseNotes/2_0_22.md

```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### Rasterize
+
+- Fixed an issue in which calling `rasterize` with URLs that started with `mailto:` would cause the script to timeout.
```

Comment:
```suggestion
- Fixed an issue in which calling `rasterize` with URLs that started with `mailto:` would cause the script to timeout; an error will be thrown instead.
```